hydrogym/core.py

```diff
@@ -47,12 +49,13 @@ class PDEBase(metaclass=abc.ABCMeta):
 
   def __init__(self, **config):
     self.mesh = self.load_mesh(name=config.get("mesh", self.DEFAULT_MESH))
+    self.reward_lambda = config.get("reward_lambda", 0.0)
```

Comment:
This doesn't seem to be used for anything.  Also not related to the physics modeled by the `PDEBase`.